### PR TITLE
Add report of metric data

### PR DESCRIPTION
**Pull request by Counterflowwind**
provide metric report client

What type of PR is this?
Add one of the following kinds:
/kind feature

What this PR does / why we need it:
provide metric report client

Specified Reivewers:
/assgin @liuhaoyang

### CODE REVIEW

**Comment by Effet** [comment]
> provide metric report client

provide to whom?

File: .github/workflows/ci-it.yml [hunk 1]
```diff
@@ -41,4 +41,4 @@ jobs:
       - name: Run Test
         run: |
           go test -work -cpu=2 -timeout 3s -race -coverprofile=coverage.txt -covermode=atomic ./...
-          bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+          bash <(curl -s https://codecov.io/bash)
```
Comment (liuhaoyang) [inline .github/workflows/ci-it.yml:44]: Each PR only needs to do one thing. 
Others have fixed the problem of lint failure, so you don't need to change ci-it.yml

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

**Review by liuhaoyang — CHANGES_REQUESTED** [review]
The result files generated by run `go test` are not allowed to be checked in to git , so you need to delete them
- coverage.txt
- pkg/license/license_key.txt
- pkg/swagger/ddlconv/test1.yml
- pkg/xmind/examples/xmind/a.xmind
- pkg/xmind/examples/xmind/a.xmind

**Review by liuhaoyang — COMMENTED** [review]
What we want is not to directly copy the code from opentelemetry to the erda repo. You need to use ServiceHub to register metric-report-client as a new provider. 
Some references 
- https://github.com/erda-project/erda/pull/36 
- https://github.com/erda-project/erda-infra/tree/develop/providers

**Pull request closed without merging**